src/components/BrandConfigProvider/styles/createStyles.ts

```diff
@@ -14,16 +14,19 @@ import { Theme } from '../theme/ThemeProvider';
  */
 export const createStyles: CreateStyles = (namespace, styleBuilder) => {
   const defaultStyles = (theme: Theme) => {
-    return [namespace, styleBuilder(theme)] as any;
+    return [namespace, styleBuilder(theme)] as const;
```

Comment:
Use `as const` instead to indicate that this tuple is readonly and of type `Readonly<StringLiteral<N>, NamedStyles<T>>`.